Bugsnag/Payload/BugsnagSession+Private.h

```diff
@@ -8,6 +8,10 @@
 
 #import <Bugsnag/BugsnagSession.h>
 
+#import "BSG_KSCrashReportWriter.h"
+
+#define BSG_PRIVATE __attribute__((visibility("hidden")))
```

Comment:
Can we put this in a utility header so that we can use it elsewhere?

Reply:
I agree we should, but wanted to leave that to a later PR

Maybe `BSGDefines.h` as a general purpose one?

I'm keen to avoid adding too many new headers, as we already have so many.